src/managers/default/index.js

```diff
@@ -32,7 +32,8 @@ class DefaultViewManager {
 			axis: this.settings.axis,
 			layout: this.layout,
 			width: 0,
-			height: 0
+			height: 0,
+			globalOptions: extend({}, options.settings, {disableSrcdoc: options.settings.disableSrcdoc || false})
```

Comment:
My intentions was to be able to able to pass it in as an option from toplevel like this:

```js
const rendition = book.renderTo('viewer', {
  contained: true,
  width: '100%',
  height: '100%',
  disableSrcdoc: true
})
```

But I guess axis should be something you also should be able to change from toplevel. Because wright now there is no external way to do this?